### PR TITLE
chore: Update Gaia roadmap

### PR DESCRIPTION
**Pull request by mmulji-ic**
Fixes issue https://github.com/cosmos/gaia/issues/2063

### CODE REVIEW

File: docs/roadmap/cosmos-hub-roadmap-2.0.md [hunk 1]
```diff
@@ -52,39 +52,18 @@ The upgrades aim to add features such as liquidity, economic security, usability
 - Gaia v8.0.x
```
Comment (glnro) [inline docs/roadmap/cosmos-hub-roadmap-2.0.md:52]: Missing updates for ibc and packet-forward-middleware.

File: docs/roadmap/cosmos-hub-roadmap-2.0.md [hunk 2]
```diff
@@ -52,39 +52,18 @@ The upgrades aim to add features such as liquidity, economic security, usability
...
 - Cosmos SDK v0.45
   - Minimal update with small fixes
-- Interchain Security - Required Participation of Provider Chain Validators
-  - The Cosmos solution to shared security that uses IBC Cross Chain Validation (CCV) to relay validator set composition from a Provider Chain (Cosmos Hub) to a Consumer Chain. This validator set is in charge of producing blocks on both networks using separate nodes. Misbehavior on the Consumer Chain results in slashing Provider Chain staking tokens (ATOM).
+- IBC Msg Whitelist to skip MinFee in CheckTX
 - Global Fee Module
   - Allows denoms and min-fees to be governance parameters so gas can be paid in various denoms.
   - Visible on [tgrade](https://github.com/confio/tgrade/tree/main/x/globalfee) already and enabled in [ante.go](https://github.com/confio/tgrade/blob/main/app/ante.go#L72-L92)
```
Comment (glnro) [inline docs/roadmap/cosmos-hub-roadmap-2.0.md:62]: Might want to link instead or additionally to Gaia docs

File: docs/roadmap/cosmos-hub-roadmap-2.0.md [hunk 3]
```diff
@@ -52,39 +52,18 @@ The upgrades aim to add features such as liquidity, economic security, usability
...
-  - Relayer Incentivisation so that IBC packets contain fees to pay for relayer costs.
-- Interchain Account Message Authorization Module
-  - Authentication module that authorizes any Account to create an Interchain Account on any IBC connected "Host" blockchain that has the Interchain Account IBC module.
-  - Accounts can be private key controlled users, and eventually the Gov Module and any Groups Module.
-- IBC Msg Whitelist to skip MinFee in CheckTX
-- Bech32 Prefix forwarding
-  - <https://github.com/osmosis-labs/bech32-ibc>
-- Liquidity Module Deprecation
-  - Contains forced withdraw of liquidity
+- Cosmos SDK v0.45
```
Comment (glnro) [inline docs/roadmap/cosmos-hub-roadmap-2.0.md:67]: Maybe also link to specific version of the SDK... its a tag of `v45.11`

File: docs/roadmap/cosmos-hub-roadmap-2.0.md [hunk 4]
```diff
@@ -52,39 +52,18 @@ The upgrades aim to add features such as liquidity, economic security, usability
 - Gaia v8.0.x
 - Cosmos SDK v0.45
```
Comment (glnro) [inline docs/roadmap/cosmos-hub-roadmap-2.0.md]:
```suggestion
- Cosmos SDK v0.45.11
```

File: docs/roadmap/cosmos-hub-roadmap-2.0.md [hunk 5]
```diff
@@ -52,39 +52,18 @@ The upgrades aim to add features such as liquidity, economic security, usability
...
-  - Accounts can be private key controlled users, and eventually the Gov Module and any Groups Module.
-- IBC Msg Whitelist to skip MinFee in CheckTX
-- Bech32 Prefix forwarding
-  - <https://github.com/osmosis-labs/bech32-ibc>
-- Liquidity Module Deprecation
-  - Contains forced withdraw of liquidity
+- Cosmos SDK v0.45
+- Interchain Security - Required Participation of Provider Chain Validators
+  - The Cosmos solution to shared security that uses IBC Cross Chain Validation (CCV) to relay validator set composition from a Provider Chain (Cosmos Hub) to a Consumer Chain. This validator set is in charge of producing blocks on both networks using separate nodes. Misbehavior on the Consumer Chain results in slashing Provider Chain staking tokens (ATOM).
+  - IBC 4.x
```
Comment (glnro) [inline docs/roadmap/cosmos-hub-roadmap-2.0.md]:
```suggestion
  - IBC 4.2
```

**Review by glnro — CHANGES_REQUESTED** [review]

File: docs/roadmap/cosmos-hub-roadmap-2.0.md (hunk 2 above)
Comment (mmulji-ic) [reply docs/roadmap/cosmos-hub-roadmap-2.0.md:62]: Added

File: docs/roadmap/cosmos-hub-roadmap-2.0.md (hunk 1 above)
Comment (mmulji-ic) [reply docs/roadmap/cosmos-hub-roadmap-2.0.md:52]: Added

File: docs/roadmap/cosmos-hub-roadmap-2.0.md (hunk 4 above)
Comment (mmulji-ic) [reply docs/roadmap/cosmos-hub-roadmap-2.0.md]: Updated

File: docs/roadmap/cosmos-hub-roadmap-2.0.md (hunk 5 above)
Comment (mmulji-ic) [reply docs/roadmap/cosmos-hub-roadmap-2.0.md]: Updated

File: docs/roadmap/cosmos-hub-roadmap-2.0.md (hunk 3 above)
Comment (mmulji-ic) [reply docs/roadmap/cosmos-hub-roadmap-2.0.md:67]: Added link

**Review by jtremback — APPROVED** [review]

**Pull request merged**